apps/docs/.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     // The Eslint rule still needs work, but you can
     // enable it to test things out.
     '@stylexjs/valid-styles': 'error',
+    // '@stylexjs/no-unused' : 'warn',
```

Comment:
you can leave this enabled.

Reply:
[update] after uncommenting, I notice the eslint error `Definition for rule '@stylexjs/no-unused' was not found.` starting appearing again, but I am certain it was working before as I was able to test the rule&auto-fix inside the docs folder. I attempted to rebuild multiple times but currently still doesn't work...